{"name": "test_login_fail", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to have text 'Thông tin tài khoản hoặc mật khẩu không đúng.'\nActual value: <element(s) not found> \nCall log:\nLocatorAssertions.to_have_text with timeout 5000ms\n  - waiting for locator(\".username\")", "trace": "self = <test_login.TestLogin object at 0x000002D590350AA0>, page = <Page url='https://hocvancokimngan.com/login/'>\n\n    @allure.story(\"Login không thành công\")\n    def test_login_fail(self, page):\n        lp= LoginPage(page)\n        lp.goto().login(settings.USER_EMAIL, \"pasw\")\n>       lp.assert_login_fail(\"Thông tin tài khoản hoặc mật khẩu không đúng.\")\n\ntests\\web\\test_login.py:21: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.login_page.LoginPage object at 0x000002D5916A1C10>, username = 'Thông tin tài khoản hoặc mật khẩu không đúng.'\n\n    def assert_login_fail(self, username):\n        with allure.step(\"Xác minh đăng nhập không thành công\"):\n>           expect(self.page.locator(self.lbl_username)).to_have_text(username)\nE           AssertionError: Locator expected to have text 'Thông tin tài khoản hoặc mật khẩu không đúng.'\nE           Actual value: <element(s) not found> \nE           Call log:\nE           LocatorAssertions.to_have_text with timeout 5000ms\nE             - waiting for locator(\".username\")\n\npages\\login_page.py:45: AssertionError"}, "steps": [{"name": "Mở trang chủ", "status": "passed", "start": 1755784448202, "stop": 1755784449350}, {"name": "Đi tới trang Sign in", "status": "passed", "start": 1755784449350, "stop": 1755784449439}, {"name": "Nhập thông tin đăng nhập", "status": "passed", "start": 1755784449439, "stop": 1755784449526}, {"name": "Đăng nhập", "status": "passed", "start": 1755784449526, "stop": 1755784450888}, {"name": "Xác minh đăng nhập không thành công", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to have text 'Thông tin tài khoản hoặc mật khẩu không đúng.'\nActual value: <element(s) not found> \nCall log:\nLocatorAssertions.to_have_text with timeout 5000ms\n  - waiting for locator(\".username\")\n\n", "trace": "  File \"C:\\Users\\thanh\\PycharmProjects\\auto_python_playwright_web\\pages\\login_page.py\", line 45, in assert_login_fail\n    expect(self.page.locator(self.lbl_username)).to_have_text(username)\n  File \"C:\\Users\\thanh\\PycharmProjects\\auto_python_playwright_web\\.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 19324, in to_have_text\n    self._sync(\n  File \"C:\\Users\\thanh\\PycharmProjects\\auto_python_playwright_web\\.venv\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\thanh\\PycharmProjects\\auto_python_playwright_web\\.venv\\Lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 481, in to_have_text\n    await self._expect_impl(\n  File \"C:\\Users\\thanh\\PycharmProjects\\auto_python_playwright_web\\.venv\\Lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 74, in _expect_impl\n    raise AssertionError(\n"}, "start": 1755784450888, "stop": 1755784455905}], "start": 1755784448200, "stop": 1755784455905, "uuid": "3ed299db-58e9-4676-ab89-71676aca8edc", "historyId": "5796fb86233b5d06cf9d5ae4bf3581b2", "testCaseId": "5796fb86233b5d06cf9d5ae4bf3581b2", "fullName": "tests.web.test_login.TestLogin#test_login_fail", "labels": [{"name": "feature", "value": "Authentication"}, {"name": "story", "value": "Login không thành công"}, {"name": "severity", "value": "critical"}, {"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "tests.web"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Thanh-PC"}, {"name": "thread", "value": "6600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.web.test_login"}]}